Tests/Serilog.Exceptions.Test/Destructurers/SocketExceptionDestructurerTest.cs

```diff
@@ -0,0 +1,36 @@
+namespace Serilog.Exceptions.Test.Destructurers
+{
+    using System.Net.Sockets;
+    using Xunit;
+    using static LogJsonOutputUtils;
+
+    public class SocketExceptionDestructurerTest
+    {
+        [Fact]
+        public void SocketException_SocketErrorCodeIsAttachedAsProperty()
+        {
+#pragma warning disable CA2208 // Instantiate argument exceptions correctly
```

Comment:
Are you sure you are getting the warning `CA2208`? Can these lines to supress this warning be removed?

Reply:
Good catch.  I removed them.